nmtwizard/preprocess/consumer.py

```diff
@@ -177,9 +180,9 @@ def _consume(self, tu_batch):
         tu_list, _ = tu_batch
         for tu in tu_list :
             if source_learner is not None:
-                _ingest_tokens(source_learner, tu.src_tok)
+                _ingest_tokens(source_learner, tu.src)
```

Comment:
Maybe rename TU to something else in the consumers to avoid confusion with TranslationUnit ?

Reply:
What do you think of `output`?

Reply:
Good ! :)